js/apps/admin-ui/src/user/UserCredentials.tsx

```diff
@@ -329,6 +330,26 @@ export const UserCredentials = ({ user }: UserCredentialsProps) => {
     }
   };
 
+  const useFederatedCredentials = user.federationLink || user.origin;
+  const [credentialTypes, setCredentialTypes] = useState<string[]>([]);
+
+  useFetch(
+    () => adminClient.users.getUserStorageCredentialTypes({ id: user.id! }),
+    setCredentialTypes,
+    []
```

Comment:
It's not likely to change, but since this fetch passes the user's id it should be a dependency here.
```suggestion
    [user.id]
```

Reply:
@jonkoops Done.